### ISSUE TITLE
Object reference not set to an instance of an object 2.10.2.0 (OutlookGoogleCalendarSync.Obfuscate.ApplyRegex)

### ISSUE THREAD

**Issue by hillcountryfare** [phw198/OutlookGoogleCalendarSync#1725]
> Checklist:-
> * [x] [Upgrade](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Upgrading) to the latest release (alpha if available) ![latest release](https://img.shields.io/github/release/phw198/OutlookGoogleCalendarSync/all.svg)
> * [x] Reproduce the issue
> * [x] Complete the information below
> * [x] Provide your `OGcalsync.log` file (see [how-to](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file))

**OGCS Version**: v2.10.2.0
**Edition**: Installed / Portable  
**Sync Direction**: O->G

**Bug Description**  
Every time sync runs get "object reference not set to an instance of an object"
I believe this is the relevant section of log:
```
2023-10-29 18:37:26,843 ERROR  11 OutlookGoogleCalendarSync.Console [292] -  Unable to reclaim orphan calendar entries in Google calendar.
2023-10-29 18:37:27,579 ERROR  11 OutlookGoogleCalendarSync.OGCSexception [28] -  System.NullReferenceException: Object reference not set to an instance of an object.
2023-10-29 18:37:27,581 ERROR  11 OutlookGoogleCalendarSync.OGCSexception [21] -  Code: 0x80004003,-2147467261; Location: ApplyRegex() in Obfuscate.cs:98:25
2023-10-29 18:37:27,582 ERROR  11 OutlookGoogleCalendarSync.OGCSexception [65] -     at OutlookGoogleCalendarSync.Obfuscate.ApplyRegex(Property property, String source, String target, Direction direction) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\Obfuscate.cs:line 98
   at OutlookGoogleCalendarSync.GoogleOgcs.Calendar.SignaturesMatch(String sigEv, String sigAi) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\GoogleOgcs\GoogleCalendar.cs:line 1921
   at OutlookGoogleCalendarSync.GoogleOgcs.Calendar.ReclaimOrphanCalendarEntries(List`1& gEvents, List`1& oAppointments, Boolean neverDelete) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\GoogleOgcs\GoogleCalendar.cs:line 1232
   at OutlookGoogleCalendarSync.Sync.Engine.Calendar.synchronize() in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\Sync\Calendar.cs:line 471
   at OutlookGoogleCalendarSync.Sync.Engine.Calendar.<>c__DisplayClass11_1.<StartSync>b__0(Object o, DoWorkEventArgs args) in C:\Users\Paul\Git\OutlookGoogleCalendarSync\src\OutlookGoogleCalendarSync\Sync\Calendar.cs:line 129
2023-10-29 18:37:27,584 ERROR  11 OutlookGoogleCalendarSync.Console [292] -  Object reference not set to an instance of an object.
```

**Steps to Reproduce the Issue**  
1. Install 2.10.2.0
2. Sync

**Any other information**  
Any additional information that may help troubleshoot the issue.  
Will provide full log file over email.

**Comment by phw198** [comment]
Thanks for reporting!

:package: This should now be resolved with [hotfix v2.10.2.1.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/13222322/v2.10.2.1.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)).

Please let me know how it goes.

**Comment by hillcountryfare** [comment]
installed a few hours ago and so far it's been stable.  I'm good closing.
